build/apis/githubapi-types.d.ts

```diff
@@ -78,16 +65,30 @@ export interface StatusEvent {
 
 // Github API -----------------------------------------------------------------
 // These provide the current bare minimum definitions for child Procbots working with them.
+export interface Comment {
+    body: string;
+    user: {
+        login: string;
+        type: string;
+    };
+    created_at: string;
+    updated_at: string;
```

Comment:
Ah ok, so if you wanted to go down that route you would have to create an entire layer of richer data structures on top, got it. Ignore me then 👍